service/service.go

```diff
@@ -128,6 +128,11 @@ func New(ctx context.Context, set Settings, cfg Config) (*Service, error) {
 	srv.telemetrySettings.MeterProvider = srv.telemetryInitializer.mp
 	srv.telemetrySettings.TracerProvider = srv.telemetryInitializer.tp
 
+	// validate the configuration
+	if err := validate(ctx, set, cfg, srv.telemetrySettings); err != nil {
+		return nil, err
+	}
```

Comment:
Would we not find out about this error anyway when we call `graph.Build`?

Reply:
Yes, that's true.  I was trying to validate it even earlier than that but you're right — it doesn't really matter.  Removing this call, thanks for catching it.

Reply:
Removed in 8729b39daebf2056602e2a9016b6cf714266ff21.